src/components/pages/events/EventsList.tsx

```diff
@@ -318,188 +215,71 @@ const EventsList = () => {
-                      ))}
-                    </div>
-                  </div>
-                </div>
-              ))
-            : upcomingEvents?.data.map((data, index) => (
+              <div className="flex flex-col gap-y-24d">
                 <div
                   className={clsx(
-                    "w-full flex flex-col gap-y-36d relative",
-                    "cursor-follow-active"
+                    "font-span md:landscape:text-48d leading-none font-bold",
+                    `event-name-${index}`
                   )}
-                  key={data.documentId}
                 >
-                  <div className="w-full h-344d rounded-32d relative overflow-hidden">
-                    <div className="px-22d py-12d rounded-32d border-[1px] border-ruby-red bg-ivory text-ruby-red uppercase absolute top-20d left-20d z-10  text-12d font-sans font-semibold">
-                      {getEventStatus(data.date, data.EndDate)}
-                    </div>
-
-                    {data.file.mime?.startsWith("image/") ? (
-                      <GradientImage
-                        src={process.env.NEXT_PUBLIC_STRAPI_URL + data.file.url}
-                        alt="collaboration image"
-                        className={clsx(
-                          "w-full h-full object-cover",
-                          `upcoming-image-index-${index}`
-                        )}
-                      />
-                    ) : (
-                      <HoverVideo
-                        src={process.env.NEXT_PUBLIC_STRAPI_URL + data.file.url}
-                        className={clsx(
-                          "w-full h-full object-cover scale-105",
-                          `image-index-${index}`
-                        )}
-                      />
-                    )}
-
-                    {data.logo && (
-                      <div className="absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2">
-                        <div className="relative w-217d h-217d">
-                          <GradientImage
-                            src={
-                              process.env.NEXT_PUBLIC_STRAPI_URL + data.logo.url
-                            }
-                            alt="collaboration image"
-                            className="h-full object-contain"
-                          />
-                        </div>
-                      </div>
-                    )}
-                  </div>
+                  {data.eventName}
+                </div>
 
-                  <div className="flex flex-col gap-y-24d">
+                <div className="flex gap-x-20d">
+                  {data.buttons.map((button, i) => (
                     <div
                       className={clsx(
-                        "font-span md:landscape:text-48d leading-none font-bold",
-                        `upcoming-event-name-${index}`
+                        "py-8d px-12d rounded-32d border-[1px] border-ruby-red font-sans text-10d",
```

Comment:
```suggestion
                        "py-8d px-12d rounded-32d border border-ruby-red font-sans text-10d",
```

sepertinya gini aja bisa ini gung, biar rapi